internal/phonehome/phonehome.go

```diff
@@ -176,11 +180,30 @@ func (phm *PhoneHomeData) fillHazelcastMetrics(cl client.Client) {
 		phm.UserCodeDeployment.addUsageMetrics(&hz.Spec.UserCodeDeployment)
 		createdMemberCount += int(*hz.Spec.ClusterSize)
 		executorServiceCount += len(hz.Spec.ExecutorServices) + len(hz.Spec.DurableExecutorServices) + len(hz.Spec.ScheduledExecutorServices)
+
+		cid, ok := ClusterUUID(hzClientRegistry, hz.Name, hz.Namespace)
+		if !ok {
+			continue
+		}
+		clusterUUIDs = append(clusterUUIDs, cid)
```

Comment:
This is a super small neat, and feel free to ignore it if you think that it doesn't make sense
```suggestion
		if !ok {
			continue
		}
		clusterUUIDs = append(clusterUUIDs, cid)
```
```suggestion
		if ok {
			clusterUUIDs = append(clusterUUIDs, cid)
		}
```